apptail/instance.go

```diff
@@ -79,10 +95,16 @@ func (instance *Instance) tailFile(name, filename string, stopCh chan bool) {
 		MaxLineSize: GetConfig().MaxRecordSize,
 		MustExist:   true,
 		Follow:      true,
-		Location:    &tail.SeekInfo{-limit, os.SEEK_END},
+		Location:    location,
 		ReOpen:      false,
 		Poll:        false,
 		RateLimiter: rateLimiter})
+
+	// IMPORTANT: this registration happens everytime app restarts
+	if shouldInitialize {
+		tracker.InitializeChildNode(instance.DockerId, filename, t.Location.Offset)
```

Comment:
Err, is `t.Location` the passed in `location` here, which has a `.Offset` of `-limit`?  (It's not using `t.Tell()` here?)